### PR TITLE
SVSR requirements

### PR DESCRIPTION
**Pull request by Glamhoth**

### CODE REVIEW

File: runtime_tests/test_demosat2.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+# \* Auxiliary LED "enginemocks"/"data indicators" driven via PIO
+# \* Auxiliary “deployment”servos driven via PIO + PWM.
+#
+# \SRS ETB-OPER-10
+def test_ProvideDemonstrationApplication():
+  pass
+
+##
+# Date: 2022.12.09
+# Result: Mission scenario exercises all high-level SEDS functions:
```
Comment (Lurkerpas) [inline runtime_tests/test_demosat2.py]: There is no scenario at this moment :/ So this cannot be covered.

File: runtime_tests/test_demosat2.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+# \* Light Sensor (sensor)
+# \* LED mocks (actuator)
+#
+# ETB-OPER-20
+def test_ExerciseAllHighLevelSedsFunctions():
+  pass
+
+##
+# Date: 2022.12.09
+# Result: Mission scenario exercises additional TASTE Functions (ModeManager and Housekeeping) and communication
```
Comment (Lurkerpas) [inline runtime_tests/test_demosat2.py]: There is no scenario at this moment.

File: runtime_tests/test_demosat2.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+#
+# \SRS ETB-OPER-10
+def test_ProvideDemonstrationApplication():
+  pass
+
+##
+# Date: 2022.12.09
+# Result: Mission scenario exercises all high-level SEDS functions:
+# \* LIDAR (sensor, but with integrated actuator)
+# \* Servos (actuator)
```
Comment (Lurkerpas) [inline runtime_tests/test_demosat2.py]: There are no servos

File: runtime_tests/test_demosat2.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+##
+# DemoSat2 SAMV71-based node contains 2 sensors - LIDAR and light sensor - described using SEDS models.
+#
+# \SRS ETB-DES-90
+def test_DemoIncludesTwoSedsDescribedSensors():
+  pass
+
+##
+# DemoSat2 SAMV71-based node contains 2 actuators - servos and LEDs - described using SEDS models.
```
Comment (Lurkerpas) [inline runtime_tests/test_demosat2.py]: There are no servos. But there is a motor in the LIDAR

File: runtime_tests/test_demosat2.py [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+#
+# \SRS ETB-DES-120
+def test_DemoIncludesLinuxBasedNode():
+  pass
+
+##
+# DemoSat2 Linux-based node contains 3 interfaces for communication with SAMV71-based node:
+# \* tc
+# \* tm
+# \* hk
```
Comment (Lurkerpas) [inline runtime_tests/test_demosat2.py:81]: 4. Also debug_hk

File: runtime_tests/test_samv71_runtime.py [hunk 6]
```diff
@@ -124,3 +124,146 @@ def test_samv71_ada_og(taste_project):
     # Check expected compilation output
     stderr = build.stderr.decode('utf-8')
     assert build.returncode == 0, 'Compilation errors: \n{}'.format(stderr)
+
+##
+# TASTE SAMV71 runtime supports deployment of TASTE models containing at least 8 TASTE functions (including
+# at least 4 EDS based). This is validated by the demonstration application (DemoSat2) which contains 14 EDS functions
```
Comment (Lurkerpas) [inline runtime_tests/test_samv71_runtime.py]: 8 EDS functions (see ADD). The rest are C-based, 13 in total.

File: runtime_tests/test_samv71_runtime.py [hunk 7]
```diff
@@ -124,3 +124,146 @@ def test_samv71_ada_og(taste_project):
...
+
+##
+# As per chapter 5.1 of ETB-N7S-ADD-001, TASTE SAMV71 runtime uses Sdramc driver to configure SDRAM attached to the MCU.
+# \SRS  ETB-DES-12
+def test_TasteSamv71RuntimeUsesSdram():
+    pass
+
+##
+# TASTE SAMV71 runtime shall use DMA in UART implementation as specified in ETB-N7S-ADD-001 document.
+# As per chapter 5.1 of ETB-N7S-ADD-001, TASTE SAMV71 runtime uses Uart driver to configure UART attached to the MCU.
```
Comment (Lurkerpas) [inline runtime_tests/test_samv71_runtime.py]: No, please refer to the fact, that we use XDMAC in UART communication.

File: runtime_tests/test_demosat2.py (hunk 1 above)
Comment (Glamhoth) [reply runtime_tests/test_demosat2.py]: That's why there is not a `\SRS` tag (so it's not counted in the requirement matrix) , I just didn't want to delete what's already has been done

File: runtime_tests/test_demosat2.py (hunk 2 above)
Comment (Glamhoth) [reply runtime_tests/test_demosat2.py]: Same as above

File: runtime_tests/test_demosat2.py (hunk 3 above)
Comment (Glamhoth) [reply runtime_tests/test_demosat2.py]: removed

File: runtime_tests/test_demosat2.py (hunk 4 above)
Comment (Glamhoth) [reply runtime_tests/test_demosat2.py]: changed

File: runtime_tests/test_demosat2.py (hunk 5 above)
Comment (Glamhoth) [reply runtime_tests/test_demosat2.py:81]: added

File: runtime_tests/test_samv71_runtime.py (hunk 6 above)
Comment (Glamhoth) [reply runtime_tests/test_samv71_runtime.py]: changed

File: runtime_tests/test_samv71_runtime.py (hunk 7 above)
Comment (Glamhoth) [reply runtime_tests/test_samv71_runtime.py]: changed

File: runtime_tests/test_demosat2.py (hunk 1 above)
Comment (Lurkerpas) [reply runtime_tests/test_demosat2.py]: It might be misleading though. Please remove.

File: runtime_tests/test_demosat2.py (hunk 2 above)
Comment (Lurkerpas) [reply runtime_tests/test_demosat2.py]: Please remove.

File: runtime_tests/test_samv71_runtime.py [hunk 8]
```diff
@@ -127,8 +127,8 @@ def test_samv71_ada_og(taste_project):
 
 ##
 # TASTE SAMV71 runtime supports deployment of TASTE models containing at least 8 TASTE functions (including
-# at least 4 EDS based). This is validated by the demonstration application (DemoSat2) which contains 14 EDS functions
-# and 1 Linux functions.
+# at least 4 EDS based). This is validated by the demonstration application (DemoSat2) which contains 8 EDS functions,
+# 5 C-based functions and 1 Linux function.
```
Comment (Lurkerpas) [inline runtime_tests/test_samv71_runtime.py]: Linux function is not deployed onto SAMV71 runtime. So it is better to write 8... + 5.... deployed onto TASTE SAMV71 Runtime.

File: runtime_tests/test_samv71_runtime.py [hunk 9]
```diff
@@ -175,7 +175,7 @@ def test_TasteSamv71RuntimeUsesSdram():
 
 ##
 # TASTE SAMV71 runtime shall use DMA in UART implementation as specified in ETB-N7S-ADD-001 document.
-# As per chapter 5.1 of ETB-N7S-ADD-001, TASTE SAMV71 runtime uses Uart driver to configure UART attached to the MCU.
+# As per chapter 5.1 of ETB-N7S-ADD-001, TASTE SAMV71 runtime uses Xdmac driver t configure DMA for UART communication.
```
Comment (Lurkerpas) [inline runtime_tests/test_samv71_runtime.py]: to

File: runtime_tests/test_demosat2.py (hunk 1 above)
Comment (Glamhoth) [reply runtime_tests/test_demosat2.py]: ack, done

File: runtime_tests/test_demosat2.py (hunk 2 above)
Comment (Glamhoth) [reply runtime_tests/test_demosat2.py]: ack, done

File: runtime_tests/test_samv71_runtime.py (hunk 8 above)
Comment (Glamhoth) [reply runtime_tests/test_samv71_runtime.py]: done

File: runtime_tests/test_samv71_runtime.py (hunk 9 above)
Comment (Glamhoth) [reply runtime_tests/test_samv71_runtime.py]: done

File: runtime_tests/test_demosat2.py (hunk 2 above)
Comment (Lurkerpas) [reply runtime_tests/test_demosat2.py]: Not removed though...

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**